content/changelog/2024-02-07-clever-tools-3.2.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Clever Tools 3.2.0 is available
+date: 2024-02-07
+tags:
+  - clever-tools
+  - cli
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Hubert Sablonière
+    link: https://github.com/hsablonniere
+    image: https://github.com/hsablonniere.png?size=40
+description: And more is coming!
+excludeSearch: true
+---
+
+As mentionned [in the previous release](/changelog/2024-01-25-clever-tools-3.1.0/), the packaging process moved to GitHub Actions to provide a better process, quicker tests and more frequent releases. We start with [the 3.2.0](https://github.com/CleverCloud/clever-tools/releases/tag/3.2.0) adding the log auto retry feature on network failures and some fixes:
+
+* **api:** improve error message with `EAI_AGAIN` and `ECONNRESET` ([b134213](https://github.com/CleverCloud/clever-tools/commit/b134213f30d46dd7f5690a38425deb4fd752148c))
+* delete beta/Jenkins mentions in README.md ([cb32b3b](https://github.com/CleverCloud/clever-tools/commit/cb32b3b0cdaecac5d6198f295eef7c350161ac03))
```

Comment:
suggestion: we can remove this I think

Reply:
Hi @davlgd 👋🏻 
Do you want to make this edit ?